.vscode/settings.json

```diff
@@ -37,4 +37,6 @@
     "quarto"
   ],
   "cSpell.language": "en,en-GB",
+  "python.testing.pytestEnabled": true,
+  "python.testing.pytestPath": "poetry run pytest",
```

Comment:
I'm not sure/I wasn't aware that you could add a command like `"poetry run pytest"` as a `pytestPath`?

Reply:
That is a great question... It seems to work since I tested it out... but now I'm not sure if it actually worked correctly or not...